roles/multiqc/tasks/main.yml

```diff
@@ -1,7 +1,20 @@
 ---
+- name: Fetch MultiQC from GitHub
+  git: repo="{{ multiqc_repo }}"
+       dest="{{ multiqc_dest }}"
+       version="{{ multiqc_version }}"
+       force=yes
+  tags: multiqc
+
+
+#- name: Install MultiQC from pip
```

Comment:
It's a bit back and forth. To quote the developer he'll be implementing a bunch of small fixes for the Irma env, so using pip is a bit too rigid. Ideally we'll go back to pip at some point.